src/applications/gi/components/EstimateYourBenefitsSummarySheet.jsx

```diff
@@ -1,96 +1,141 @@
 import React from 'react';
-import { CalculatorResultRow as ImportedCalculatorResultRow } from '../components/profile/EstimatedBenefits';
+import classNames from 'classnames';
 
-const CalculatorResultRow = ({ label, value, header, bold, visible }) => (
-  <ImportedCalculatorResultRow
-    id={`summary-sheet-${label}`}
-    label={label}
-    value={value}
-    header={header}
-    bold={bold}
-    visible={visible}
-  />
-);
+const CalculatorSheetResultRow = ({
+  id,
+  label,
+  value,
+  header,
+  visible,
+  bold,
+  boldLabel,
+  plainTextValue,
+}) =>
+  visible ? (
+    <div
+      id={`calculator-result-row-${id == null ? label : id}`}
```

Comment:
```suggestion
      id={`summary-sheet-calculator-result-row-${id == null ? label : id}`}
```
otherwise e2e tests will fail